packages/sw-cache-expiration/README.md

```diff
@@ -1,5 +1,8 @@
 <!-- DO NOT EDIT. This page is autogenerated. -->
 <!-- To make changes, edit templates/Project-README.hbs, not this file. -->
+
+All of the modules are described below. Each has an _About_ page with basic usage instructions and a _Demo_ directory with an example. The main page of the documentation is [here](https://googlechrome.github.io/sw-helpers/#main).
```

Comment:
This is the packages readme, so there is only one module being referenced so the intro "All of the modules are described below" doesn't make sense in this context.